src/home/github-types.ts

```diff
@@ -55,75 +55,23 @@ export interface GitHubUserResponse {
   data: GitHubUser;
 }
 
-interface GitHubLabel {
-  id: number;
-  node_id: string;
-  url: string;
-  name: string;
-  description: string;
-  color: string;
-  default: boolean;
-}
-
-interface GitHubMilestone {
-  url: string;
-  html_url: string;
-  labels_url: string;
-  id: number;
-  node_id: string;
-  number: number;
-  state: string;
-  title: string;
-  description: string;
-  creator: GitHubUser;
-  open_issues: number;
-  closed_issues: number;
-  created_at: string;
-  updated_at: string;
-  closed_at: string | null;
-  due_on: string | null;
-}
-
-export interface GitHubIssue {
-  id: number;
-  node_id: string;
-  url: string;
-  repository_url: string;
-  labels_url: string;
-  comments_url: string;
-  events_url: string;
-  html_url: string;
-  number: number;
-  state: string;
-  title: string;
-  body: string;
-  user: GitHubUser;
-  labels: GitHubLabel[];
-  assignee: GitHubUser | null;
-  assignees: GitHubUser[];
-  milestone: GitHubMilestone | null;
-  locked: boolean;
-  active_lock_reason: string | null;
-  comments: number;
-  pull_request?: {
-    url: string;
-    html_url: string;
-    diff_url: string;
-    patch_url: string;
-  };
-  closed_at: string | null;
-  created_at: string;
-  updated_at: string;
-  closed_by?: GitHubUser;
-}
+export type GitHubIssue = RestEndpointMethodTypes["issues"]["get"]["response"]["data"];
 
 export interface AvatarCache {
   [organization: string]: string | null;
 }
 
 export const GITHUB_TASKS_STORAGE_KEY = "gitHubTasks";
 
+// supabase key should be dynamic incase of change and testing
+let supabaseUrl = "";
```

Comment:
```suggestion
let supabaseUrl;
```

Reply:
This is only used once and it needs to be a string so the substring method don't error out

Reply:
And for the supabase url, it is already throwing an error if emoty somewhere else so it won't run if its empty anyway

Reply:
I think its better `supabaseUrl` that's not exported is a string than for the exported `SUPABASE_STORAGE_KEY` to return a string